android/src/main/kotlin/com/icapps/background_location_tracker/service/LocationUpdatesService.kt

```diff
@@ -173,6 +185,7 @@ internal class LocationUpdatesService : Service() {
             wakeLock?.release();
         }
         Logger.debug(TAG, "Removing location updates")
+        val locationCallback = this.locationCallback ?: return
```

Comment:
Same for this

Reply:
removeLocationUpdates requires a non nullable value